src/foam/nanos/auth/User.js

```diff
@@ -239,6 +239,7 @@ foam.CLASS({
       },
       view: { class: 'foam.u2.detail.VerticalDetailView' },
       createVisibility: 'HIDDEN',
+      hidden: true,
```

Comment:
replace both createVisibility and hidden with 
visibility: 'HIDDEN'
here and elsewhere.